index.js

```diff
@@ -30,7 +31,15 @@ Gulp.prototype.watch = function(glob, opt, task) {
     fn = this.parallel(task);
   }
 
-  return vfs.watch(glob, opt, fn);
+  var watcher = chokidar.watch(glob, opt);
+  if (fn) {
```

Comment:
@es128 the opt argument is 100% different now that the options are from chokidar and not gaze so those options need to be documented